drivers/mpu9250/MPU9250.cpp

```diff
@@ -250,8 +312,26 @@ int MPU9250::mpu9250_deinit()
 	return 0;
 }
 
+int MPU9250::pinThread(int cpu)
```

Comment:
In order to obtain the lowest latency, pinning the thread processing data (in this case SPI), the kernel worker thread (kernel spi thread) and assigning IRQ affinity is the only option on Linux.
I don't see why this would hurt if you give the user the ability to pin to a specific thread when he starts the driver.